src/lib.rs

```diff
@@ -19,7 +19,7 @@ fn test_results_parser(py: Python, m: &Bound<PyModule>) -> PyResult<()> {
     m.add_function(wrap_pyfunction!(junit::parse_junit_xml, m)?)?;
     m.add_function(wrap_pyfunction!(failure_message::build_message, m)?)?;
     m.add_function(wrap_pyfunction!(
-        failure_message::escape_failure_message,
+        failure_message::escape_message,
```

Comment:
I don't believe anyone uses this right now, so renaming it should be fine... (I'm also considering removing this generally from the interface).

Reply:
indeed it should be fine to rename it, but if it's removed from the interface then how will the function be called by the CLI / worker?

Reply:
This function currently doesn't get called in the worker/CLI. Worker doesn't use this and CLI uses `build_message` (which calls this). So, I don't think we need to expose it externally at the moment.

Reply:
ah sorry i got confused, yeah i think it makes sense to remove this from the interface